test/integration/authz_v1_test.go

```diff
@@ -110,7 +111,7 @@ func TestCreateServiceRoleAndBinding(t *testing.T) {
 }
 
 // 1.1 Create SR only if there are no role members for SRB creation
```

Comment:
let's update the comment - `create SRB with role cert spiffe only if there are no role members`

Reply:
Updated the comment